aten/src/ATen/cuda/CuSparseDescriptors.h

```diff
@@ -0,0 +1,126 @@
+#pragma once
+
+// guard the whole file
+#if !defined(_MSC_VER) && defined(__CUDACC__) && CUDART_VERSION >= 10010 // CUDA release >= 10.1 and not windows
```

Comment:
@xwang233 please use something like && defined(CUSPARSE_VER_MAJOR) && CUSPARSE_VER_MAJOR >= 10 to detect CUDA-10.1 Update 1 or newer

Reply:
Thanks for the comments! I'll check that with different library versions.